blocks/feed/feed.js

```diff
@@ -100,11 +110,14 @@ function getMetadataNullable(key) {
  * Feed block decorator to build feeds based on block configuration
  */
 export default async function decorate(block) {
+  let blockContents = 0;
+  let queryObj = 0;
   const blockCfg = readBlockConfig(block);
   const blockName = (blockCfg['block-type'] ?? 'cards').trim().toLowerCase();
   const blockType = (blockName.split('(')[0]).trim();
   const variation = (blockName.match(/\((.+)\)/) === null ? '' : blockName.match(/\((.+)\)/)[1]).trim();
-  const queryObj = await queryIndex(`${getLanguage()}-search`);
+  if (`${getLanguage()}` === 'ja') queryObj = await queryIndex('jp-search');
```

Comment:
`getLanguage` should take care of language defaults and `ja/jp` etc.

Reply:
Yes I was getting ja as the output of getLanguage, that's why. Seems like Today Ashutosh has done some changes to make the getLanguage - - -> resulting in "ja". May be once merged to main, will remove this check